rust/connlib/tunnel/src/tests.rs

```diff
@@ -45,9 +45,6 @@ use tracing_subscriber::{util::SubscriberInitExt as _, EnvFilter};
 
 proptest_state_machine::prop_state_machine! {
     #![proptest_config(Config {
-        // Enable verbose mode to make the state machine test print the
-        // transitions for each case.
-        verbose: 1,
         cases: 1000,
```

Comment:
You can also control it with an env variable `PROPTEST_CASES`. It will run all previously known bad seeds first so whilst debugging a case, there is not really a need to adjust it.

The default is 256 I believe. We could leave it at that and set it via the env variable in CI?

Reply:
> .]It will run all previously known bad seeds first so whilst debugging a case, there is not really a need to adjust it.

The number of shrink iters is based on the number of cases(I think x4 times) so that's the important thing when debuging, but I didn't realize I can control it with the env variable.

It's better to override it locally than in CI I think, so let's leave it like this :)